README.md

```diff
@@ -90,6 +90,7 @@ CLUSTERS = [
         cluster="paas-prod",
         location="us-west1",
         project="cruise-paas-prod",
+        # use_internal_ip="true", # (optional, default use public endpoint)
```

Comment:
remove the leading `#` looks better, and maybe the comment should be `default to "false", which uses public endpoint`

Reply:
ohh, kk, on it